frontend/packages/policy-editor/src/utils/PolicyEditorUtils.test.ts

```diff
@@ -1,3 +1,4 @@
+import { PolicyAction, PolicyRule, PolicySubject } from '@altinn/policy-editor';
```

Comment:
As we are inside the package ```@altinn/policy-editor```, the imported code should use the absolute path to where the imported code is coming from. 
Something like this: 
```suggestion
import { PolicyAction, PolicyRule, PolicySubject } from '../types';
```